source-hubspot-native/tests/snapshots/source_hubspot_native_tests_test_snapshots__capture__stdout.json

```diff
@@ -1,5574 +1,2081 @@
 [
   [
-    "acmeCo/companies",
+    "acmeCo/marketing_emails",
     {
+      "activeDomain": "45386308.hs-sites.com",
```

Comment:
There's a lot changing in this snapshot. Understandably the new resources will add to it, but the churn on the pre-existing outputs is hard to digest. Is any of the existing output changing, or is just all from shifting the ordering of the things around?

Reply:
Lots of shifiting around since this snapshot was first added when we had only 4 streams